.idea/codeStyles/Project.xml

```diff
@@ -3,6 +3,115 @@
     <JetCodeStyleSettings>
       <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
     </JetCodeStyleSettings>
+    <codeStyleSettings language="XML">
```

Comment:
@pritamkarmakar i think you need to go over your git-ignore. this file shouldn't be included.

Reply:
@adrianaisemberg thank you, updated gitignore and also deleted few files in latest commit.